### PR TITLE
New toggle

### PR DESCRIPTION
**Pull request by dflippojr**
working implementation of list toggle, all that's left is a discussion of looks and style

### CODE REVIEW

File: src/highlighty.js [hunk 1]
```diff
@@ -82,14 +82,17 @@ $(function () {
...
-      log('highlightPhrases ' + phraseListIndex);
+      log('highlightPhrases ' + phraseListIndex + ' toggled:' + options.highlighter[phraseListIndex].toggled);//the added toggle here is undefined
       let markClasses = `${HL_BASE_CLASS} ${HL_PREFIX_CLASS}${phraseListIndex}`;
       let hilitor = new Hilitor();
-      hilitor.applyPhrases(phrasesToHighlight[phraseListIndex], {
-        classes: markClasses,
-        caseSensitive: !options.enableCaseInsensitive,
-        partialMatch: options.enablePartialMatch,
-      });
+      let toggled = options.highlighter[phraseListIndex].toggled;
```
Comment (wustep) [inline src/highlighty.js:88]: Prettier

File: src/highlighty.js [hunk 2]
```diff
@@ -82,14 +82,17 @@ $(function () {
   function highlightPhrases(options) {
     log('highlightPhrases start');
     for (let phraseListIndex in phrasesToHighlight) {
-      log('highlightPhrases ' + phraseListIndex);
+      log('highlightPhrases ' + phraseListIndex + ' toggled:' + options.highlighter[phraseListIndex].toggled);//the added toggle here is undefined
```
Comment (wustep) [inline src/highlighty.js:85]: get rid of comment

File: src/highlighty.js (hunk 1 above)
Comment (wustep) [reply src/highlighty.js:88]: One option for backwards compatibility is.. you can do 

`options.highlighter[phraseListIndex].toggled === true || options.highlighter[phraseListIndex].toggled === undefined`

File: src/options.html [hunk 3]
```diff
@@ -175,9 +175,17 @@ <h2 class="title">Lists</h2>
             <div class="column is-three-fifths">
               <div class="field is-grouped PhraseList__header">
                 <h1 class="title">
-                  <button class="PhraseList__color" style="background-color: #000000"></button>
+                  <input
+                    id="PhraseList__toggle"
```
Comment (wustep) [inline src/options.html:179]: 1. let's grid of this id 
2. rename `input` below -> `toggle`

File: src/options.html [hunk 4]
```diff
@@ -175,9 +175,17 @@ <h2 class="title">Lists</h2>
...
+                    type="checkbox"
+                    class="switch is-outlined is-medium is-info PhraseList__input"
+                    checked="checked"
+                  />
+                  <label class = "PhraseList__label" for="PhraseList__toggle">
```
Comment (wustep) [inline src/options.html:184]: 1. we can just drop the id above, and then fill it in
2. let's rename `PhraseList__input` -> `PhraseList__toggle`
3. does this label do anything? if it's needed for styling purposes, that's fine let's just call this `PhraseList__toggleLabel`

**Review by wustep — APPROVED** [review]

**Review by wustep — APPROVED** [review]

**Comment by dflippojr** [comment]
https://github.com/wustep/highlighty/assets/93831415/01eb4959-113a-426e-a6b4-408f10e9dafd

**Review by wustep — APPROVED** [review]

**Pull request merged**